internal/cloud/realm/auth_test.go

```diff
@@ -78,16 +78,46 @@ func TestRealmAuthRefresh(t *testing.T) {
 		assert.Equal(t, nil, err)
 
 		// invalidate the session's tokens
-		session.AccessToken = ""
 		session.RefreshToken = session.AccessToken
+		session.AccessToken = ""
 
 		profile := mock.NewProfileWithSession(t, session)
 
 		client = realm.NewAuthClient(profile.RealmBaseURL(), profile)
 		_, err = client.AuthProfile()
 		assert.Equal(t, realm.ErrInvalidSession{}, err)
 	})
-	// TODO: REALMC-7719 add test for expired credentials and test for ensuring profile cleared on invalid session
```

Comment:
I am comfortable with the scope of these tests now. We can use a non-mocked profile if we actually want to check that the tokens get set/reset appropriately. I am fine doing that in this ticket as well